recipe/meta.yaml

```diff
@@ -10,12 +10,15 @@ source:
   sha256: 17a6f941e4fa06d08a628990f6816d1da5e545d65f533e6f598740d2cb76ace4
 
 build:
-  skip: true  # [py<39 or osx]
+  skip: true  # [py<39]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 1
 
 requirements:
   build:
+    - python  # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - cmake >=3.13  # [build_platform != target_platform]
```

Comment:
Needed to avoid `error: [Errno 86] Bad CPU type in executable: 'cmake'` in build given `cmake` is a `build` requirement binary that gets used for the `host` as well.

https://github.com/scikit-hep/pyhepmc/blob/11903d8ede68117189702581be8d61b7ccef10f5/pyproject.toml#L5

```toml
[build-system]
requires = [
    "setuptools>=46.4",
    "setuptools_scm[toml]>=6.2",
    "cmake>=3.13",
    "wheel",
]
build-backend = "setuptools.build_meta"
```

c.f. https://conda-forge.org/docs/maintainer/knowledge_base/#details-about-cross-compiled-python-packages

Reply:
BTW, packages should never list `cmake` as a build requirement. They should use a custom build hook that adds it only if needed or just use scikit-build-core.

They also should never list wheel, especially since it's no longer used.

Though that's not the problem here, as this is ignored for conda-forge, you did the correct thing which was to include cmake in the conda recipe.

Reply:
Yeah. We should probably open up an issue with https://github.com/scikit-hep/pyhepmc/ on this to avoid things like this in the future.

Reply:
Also `setuptools_scm` has an empty `[toml]` extra as of [`v8.0.0`](https://github.com/pypa/setuptools-scm/releases/tag/v8.0.0), so the https://github.com/scikit-hep/pyhepmc/ `build-system` could get updated to

```toml
[build-system]
requires = [
    "setuptools>=64",
    "setuptools_scm>=8.0.0",
]
build-backend = "setuptools.build_meta"
```